app/src/main/res/layout/bottom_sheet_scooter_start_ride.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
You can build this layout only using CL and a background shape drawable for top left & top right corners